.vuepress/config.js

```diff
@@ -81,6 +81,7 @@ module.exports = {
           children: [
             '/learn/advanced/asynchronous_updates',
             '/learn/advanced/snapshots_vs_dumps',
+            '/learn/advanced/updating.md',
```

Comment:
Not 100% certain but pretty sure that if we don't specify a title, VuePress will use the first H1 in the file, which in this case is `# Update to the latest MeiliSearch version`. Seems a bit long to me, therefore I prefer:

```suggestion
            {
              title: 'Updating MeiliSearch',
              path: '/learn/advanced/updating.md'
            },
```